README.md

```diff
@@ -188,20 +188,28 @@ Note down the client key somewhere safe, like a password vault.
 You will only ever see this credential once.
 This is your personal key, and you should not share it.
 
-Set these as environment variables in the `.envrc` file using the below variable names:
+Set these in your [Rails credentials](https://guides.rubyonrails.org/security.html#custom-credentials) file:
+
+```yaml
+cipherstash:
+  client_id:
+  client_key:
+```
+
+Or set these as environment variables in a `.envrc` file using the below variable names:
 
 ```bash
 export CS_CLIENT_KEY=
 export CS_CLIENT_ID=
 ```
 
-If you are using `direnv` run:
+If you are using direnv run:
 
 ```bash
 direnv allow
 ```
 
-If you're not you can export the variables by running:
+Run:
```

Comment:
What's the reason for removing the note about not using direnv here? The original text looks correct to me.

Alternatively, we could probably remove mentions of direnv in the demo and assume that the user knows how to set env vars. We could say "set these env vars" and let the user set them however they prefer.

Reply:
hmm, I think this may have been unintentional, as it doesn't read right. I was copying and pasting a bit over from the docs repo.

I think it's good to have it in there, trying to be helpful.

Reply:
I'll revert it back.